src/openrct2/ride/Vehicle.cpp

```diff
@@ -5204,7 +5204,14 @@ static void vehicle_kill_all_passengers(Vehicle* vehicle)
     if (numFatalities != 0)
     {
         ride->FormatNameTo(gCommonFormatArgs + 2);
-        news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
+        if (numFatalities == 1)
+        {
+			news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PERSON_DIED_ON_X, vehicle->ride);
+        }
+        else
+        {
+			news_item_add_to_queue(NEWS_ITEM_RIDE, STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
+        }
```

Comment:
```suggestion
        news_item_add_to_queue(NEWS_ITEM_RIDE, numFatalities == 1 ? STR_X_PERSON_DIED_ON_X : STR_X_PEOPLE_DIED_ON_X, vehicle->ride);
```
I prefer it like this, but you can also assign to a variable based on this and have just one call for `add_to_queue`